.github/workflows/e2e-test.yaml

```diff
@@ -120,24 +120,31 @@ jobs:
 
           namespaces=$(kubectl get namespaces -o jsonpath='{.items[*].metadata.name}')
           for namespace in $namespaces; do
-            if [[ "$namespace" == "neonvm-system" ]] || [[ "$namespace" == kuttl-test-* ]]; then
-              tee_if_needed=$GITHUB_STEP_SUMMARY
+            if [[ "$namespace" == "neonvm-system" ]] || [[ "$namespace" == kuttl-test-* ]] || [[ "$namespace" == "kube-system" ]]; then
+              tee_ns_if_needed=$GITHUB_STEP_SUMMARY
             else
-              tee_if_needed=/dev/null
+              tee_ns_if_needed=/dev/null
             fi
 
             {
               echo "<details>"
               echo "<summary>Namespace=$namespace</summary>"
-            } | tee -a $tee_if_needed
+            } | tee -a $tee_ns_if_needed
 
             pods=$(kubectl get pods -n $namespace -o jsonpath='{.items[*].metadata.name}')
             for pod in $pods; do
```

Comment:
It looks like we get all logs & events inside this for-loop. Let's move the `if` under it, and then we can have all conditions(and it won't be required to split `tee_if_needed`).

It'll require dropping the outer scoping by namespaces, but it's probably fine, we still mention namespaces for each entry.